### DISCUSSION TITLE
#help |  HA is still trying to load bubble-pop-up.js

### DISCUSSION THREAD

**Discussion by BaronTankhe** [Clooos/Bubble-Card#620] [Questions about config, custom styles and templates] [answered]
Hi,

Somehow home assistant is still trying to load https://localhost:8123/hacsfiles/Bubble-Card/bubble-pop-up.js but it is not there anymore. Does anybody have any suggestions as how to fix this?
In the console it is still showing as _Bubble Card - Pop-up  v1.6.4_

**Comment by pgale** [comment]
If you search here, you'll see several mentions of similar:

https://github.com/Clooos/Bubble-Card/discussions/489

**Reply by BaronTankhe** [reply, marked as answer]
Ah the actual text regarding the optimization is gone in the latest readme. I found it after going to an earlier tagged version.
I completely forgot I added 

```javascript
frontend:
  extra_module_url:
    - /hacsfiles/Bubble-Card/bubble-pop-up-fix.js
``` 
  to the configuration. After removing it everything works fine!!

**Comment by BaronTankhe** [comment]
Only issue remaining is all my popups now vertically align since version 2024.7 which also messed up another card sizing.....

**Reply by Clooos** [reply]
Can you open a new issue with all the details?
